app/components/topNav.tsx

```diff
@@ -0,0 +1,245 @@
+    <>
+      <div className="mx-auto bg-gray-700">
+        <div className="mx-auto max-w-7xl px-2 sm:px-6 lg:px-8 bg-gray-700">
+          <div className="relative flex h-16 items-center justify-between">
+            <div className="absolute inset-y-0 left-0 flex items-center sm:hidden">
+              {/* <!-- Mobile menu button--> */}
+              <button
+                type="button"
+                className="inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
+                aria-controls="mobile-menu"
+                aria-expanded="false"
+              >
+                <span className="sr-only">Open main menu</span>
+                {/* <!--
+              Icon when menu is closed.
+
+              Menu open: "hidden", Menu closed: "block"
+            --> */}
+                <svg
+                  className="block h-6 w-6"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  strokeWidth="1.5"
+                  stroke="currentColor"
+                  aria-hidden="true"
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+                  />
+                </svg>
+                {/* <!--
+              Icon when menu is open.
+
+              Menu open: "block", Menu closed: "hidden"
+            --> */}
+                <svg
+                  className="hidden h-6 w-6"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  strokeWidth="1.5"
+                  stroke="currentColor"
+                  aria-hidden="true"
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    d="M6 18L18 6M6 6l12 12"
+                  />
+                </svg>
+              </button>
+            </div>
+            <div className="flex flex-1 items-center justify-center sm:items-stretch sm:justify-start">
+              <div className="flex flex-shrink-0 items-center">
+                <img
+                  className="block h-8 w-auto lg:hidden"
+                  src={logo_sq}
+                  alt="Plan It Social Logo!"
+                />
+                <img
+                  className="hidden h-8 w-auto lg:block"
+                  src={logo_hori}
+                  alt="Plan It Social Logo!"
+                />
+              </div>
+              <div className="hidden sm:ml-6 sm:block">
+                <div className="flex space-x-4">
+                  {/* <!-- Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" --> */}
+                  <Link
+                    to="/"
+                    title="Home"
+                    aria-label="Home"
```

Comment:
`aria-label` overrides the text content of the element for screen readers. Here we would replace "Home" with "Home", so there is no need for the `aria-label` (same with `title`, I believe).